### PR TITLE
Created HI/Chapter1/5.mdx

### PR DESCRIPTION
**Pull request by pandyaved98**
Created HI/Chapter1/5.mdx and added same into the _toctree.yml file as well. Continuing with the next topics and chapters.

### CODE REVIEW

File: chapters/hi/_toctree.yml [hunk 1]
```diff
@@ -27,7 +40,6 @@
     title: डेटा संसाधित करना
   - local: chapter3/3
     title: मॉडल कि फाइन-ट्यूनिंग Trainer API या Keras के साथ
-    local_fw: { pt: chapter3/3, tf: chapter3/3_tf }
```
Comment (lewtun) [inline chapters/hi/_toctree.yml:30]: I think we need to revert this change in order to build the docs. Is there a reason why you needed to remove it?

**Review by lewtun — COMMENTED** [review]
Thanks for adding all these sections @pandyaved98 !! I think there's a small change needed in the `_toctree.yml` file, and then this should be good to go

File: chapters/hi/_toctree.yml (hunk 1 above)
Comment (pandyaved98) [reply chapters/hi/_toctree.yml]: The changes are reversed. Have a look and let me know. @lewtun

Comment (lewtun) [reply chapters/hi/_toctree.yml]: Thanks, this looks great!

**Review by lewtun — APPROVED** [review]
Thanks for fixing the doc build - merging this!

**Comment by pandyaved98** [comment]
The pull request is merged but didn't show up on the main page. @lewtun 

Crashed
